kit/ForKit.cpp

```diff
@@ -146,6 +146,12 @@ class ServerWSHandler final : public WebSocketHandler
             {
                 LOG_WRN("Cannot spawn " << tokens[1] << " children as requested.");
             }
+        }
+		else if (tokens.size() == 2 && tokens.equals(0, "setloglevel"))
+        {
+            // Set environment variable so that new children will also set their log levels accordingly.
+            setenv("LOOL_LOGLEVEL", tokens[1].c_str(), true);
+			Log::logger().setLevel(tokens[1]);
```

Comment:
Please don't mix tabs and spaces. All C and C++ files use spaces (and never tabs) in this project.